app/src/main/java/com/example/demorecyclerview/DragManagerAdapter.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.demorecyclerview
+
+import android.content.Context
+import android.support.v7.widget.RecyclerView
+import android.support.v7.widget.helper.ItemTouchHelper
+
+class DragManagerAdapter(adapter: ItemAdapter, context: Context, dragDirs: Int, swipeDirs: Int) :
```

Comment:
Context nên đứng đầu thứ tự các params
```suggestion
class DragManagerAdapter(context: Context, adapter: ItemAdapter, dragDirs: Int, swipeDirs: Int) :
```